pom.xml

```diff
@@ -188,6 +188,7 @@
         <module>integration-metrics</module>
         <module>integration-webhook</module>
         <module>integration-api-client</module>
+        <module>integration-authorization</module>
```

Comment:
I believe all the code implemented here are related to OAuth1, so I suggest you rename to 'integration-oauth1'.